py-notebook/jupyter-widget/js/README.md

```diff
@@ -0,0 +1,11 @@
+A jupyter widget of the hierchical confusion matrix Neo CHI 2022
+
+Package Install
+---------------
+
+**Prerequisites**
+- [node](http://nodejs.org/)
+
+```bash
+npm install --save neo-confusion-jupyter
```

Comment:
Could you say a bit more about this? I'm not as familiar with jupyter widgets, so I'm wondering who is the consumer for this potential javascript package. Can the python package not include any necessary javascript?